package.json

```diff
@@ -3,11 +3,9 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
+  "type": "module",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "npm run build -ws --if-present",
-    "eik:login": "eik login",
-    "eik:publish": "npm run eik:publish -ws"
```

Comment:
These are not needed as we can run the commands directly in the package repos via workspaces (workspaces=true set in .npmrc file)